DOC-950: Updated SQL diagrams for PARTITION ALL BY

nickvigilante opened the pull request:
Updated diagrams.go to incorporate new syntax for PARTITION BY/PARTITION ALL BY

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/78961)
<!-- Reviewable:end -->

Commit:
DOC-950: Updated SQL diagrams for PARTITION ALL BY

Updated diagrams.go to incorporate new syntax for PARTITION BY/PARTITION ALL BY

Release note: None

nickvigilante commented:
@RichardJCai Can you bors for me?

RichardJCai commented:
> @RichardJCai Can you bors for me?

Yeah will bors when green

Commit:
re-added diagrams.go

Release note: None

RichardJCai commented:
bors r+
